docs/regelwerk.mdx

```diff
@@ -132,32 +134,28 @@ Es gibt keine allgemeinen Turnierabläufe. Wie genau sich ein Turnier abspielt,
 
 1. Wenn ein Becher mit einem Ball getroffen wird, der mindestens einmal aufgesetzt hat, zählt dieser Treffer als zwei Treffer.
 2. Sobald ein Ball mindestens einmal auf dem Tisch oder einem Gegenstand auf dem Tisch aufgesetzt hat, darf er abgewehrt werden.
-3. Ein Ball zählt als Aufsetzer, sobald er mindestens einmal auf dem Tisch oder einem Gegenstand auf dem Tisch aufgesetzt hat.
+3. Ein Ball zählt als Aufsetzer und somit als zwei Treffer, sobald er mindestens einmal auf dem Tisch aufgesetzt hat.
```

Comment:
1. and 3. are very similar and can be combined imo.

e. g.
"Ein Ball, welcher mindestens einmal auf dem Tisch aufgesetzt hat und in einem Becher landet, zählt als Aufsetzer und somit als zwei Treffer."